source/directives/uiTreeNode.js

```diff
@@ -747,9 +747,7 @@
                                                       : (targetElmOffset.top + targetElmOffset.height - childsHeight - ((targetElmOffset.height - childsHeight) * scope.$treeScope.coverage));
 
                       if (elmVertUp <= upLimit) {
-                        if (targetNode.accept(scope, targetNode.index())) {
-                          targetElm[0].parentNode.insertBefore(placeElm[0], targetElm[0]);
-                        }
```

Comment:
@fadliawan are you able to justify why you are removing this conditional? From my understanding, the `accept()` methods checks whether the node is able to be dropped within the target.

Won't removing this check mean that it is able to be dropped anywhere in the tree component?